### PR TITLE
Bug/expression lost after csub cache refresh

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes issue #2202.
Possibly also #2203, as a call to httpInfo.fill() has been added in the second mongoSubCacheItemInsert.
However, some investigation is needed to ensure #2203 is fixed (a functest)

### CODE REVIEW

File: src/lib/jsonParseV2/parseSubscription.cpp [hunk 1]
```diff
@@ -582,6 +582,7 @@ static std::string parseNotifyConditionVector(ConnectionInfo* ciP, ngsiv2::Subsc
   {
```
Comment (fgalan) [inline src/lib/jsonParseV2/parseSubscription.cpp]: CNR entry is missing.

File: test/functionalTest/cases/1316_create_subscription/cond_attrs_and_q.test [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Using attrs and q at the same time in conditions
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0 IPv4 -subCacheIval 2
+#brokerStart CB 0 IPv4 -noCache
```
Comment (fgalan) [inline test/functionalTest/cases/1316_create_subscription/cond_attrs_and_q.test]: This commented line (useful while debugging) should be removed.

File: src/lib/jsonParseV2/parseSubscription.cpp (hunk 1 above)
Comment (kzangeli) [inline src/lib/jsonParseV2/parseSubscription.cpp]: Fixed in 03c62e0

File: test/functionalTest/cases/1316_create_subscription/cond_attrs_and_q.test (hunk 2 above)
Comment (kzangeli) [inline test/functionalTest/cases/1316_create_subscription/cond_attrs_and_q.test]: Fixed in 03c62e0

**Comment by fgalan** [comment]
LGTM

**Pull request merged**